Controllers/Settings.php

```diff
@@ -3,35 +3,59 @@
 namespace Leantime\Plugins\TimeTable\Controllers;
 
 use Leantime\Core\Controller;
+use Leantime\Core\Frontcontroller;
+use Symfony\Component\HttpFoundation\RedirectResponse;
 use Symfony\Component\HttpFoundation\Response;
+use Leantime\Domain\Setting\Repositories\Setting as SettingRepository;
 
 /**
- * Settings Controller for Timetable plugin
+ * Settings Controller for TimeTable plugin
  *
  * @package    leantime
  * @subpackage plugins
  */
 class Settings extends Controller
 {
+    private SettingRepository $settingsRepo;
+
     /**
-     * init
+     * constructor
+     * @access public
      *
      * @return void
      */
-    public function init(): void
+    public function init(SettingRepository $settingsRepo): void
     {
+        $this->settingsRepo = $settingsRepo;
     }
 
     /**
-     * get
+     * get method
      *
      * @return Response
-     *
      * @throws \Exception
      */
     public function get(): Response
     {
+        $ticketCacheExpiration = (int) ($this->settingsRepo->getSetting('timetablesettings.ticketscache') ?: 1200);
+
+        $this->tpl->assign('ticketCacheExpiration', $ticketCacheExpiration);
+
+        return $this->tpl->display('timeTable.Settings');
+    }
+
+    /**
+     * post method
+     * @param array<string, int> $params
+     * @return RedirectResponse
+     * @throws \Exception
+     */
+    public function post(array $params): RedirectResponse
+    {
+        $this->settingsRepo->saveSetting('timetablesettings.ticketscache', (int) ($params['ticketCacheExpiration'] ?? 0));
```

Comment:
Two things:
1. Given that we are writing to a shared repository, `timetablesettings` should probably be name spaced somehow to minimize any (even remote) risk of the setting being overwritten by either core or other plugins . E.g. `itk-leantime-timetable` 
2. We should use the same paramter name all round for the same value to minimize confusion. So if the param is `ticketCacheExpiration` then the settings key should be the same. So with (1) it should be `saveSetting('itk-leantime-timetable.ticketCacheExpiration', ...)`